portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDiscovery/HeadlessDiscovery.testcase

```diff
@@ -0,0 +1,36 @@
+@component-name = "portal-headless-frontend-infrastructure"
+definition {
+
+	property testray.main.component.name = "Headless Frontend Infrastructure";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "Load headless delivery openAPI explorer"
+	@priority = "5"
+	test CanLoadHeadlessDeliveryOpenAPI {
+		property portal.acceptance = "true";
+
+		task ("Given navigate to headless delivery endpoint") {
+			Api.gotoHeadlessDeliveryAPI();
+		}
+
+		task ("Then headless delivery api loaded with no errors") {
+			AssertTextPresent(value1 = "Headless Delivery");
```

Comment:
Maybe we could assert text present in a specific element? Maybe we could assert the text of the element that is included in the class "servers"? If we only focus on the text appearing on the screen it may not be enough to assure that the elements of the page have been loaded correctly. 
<img width="1191" alt="image" src="https://user-images.githubusercontent.com/35199951/176389955-90d60a05-3ea5-4ba0-8588-0f149377b70f.png">